crates/compiler/builtins/bitcode/src/crypt.zig

```diff
@@ -0,0 +1,40 @@
+const std = @import("std");
+const crypto = std.crypto;
+const sha2 = crypto.hash.sha2;
+const list = @import("list.zig");
+const utils = @import("utils.zig");
+
+const Sha256 = extern struct {
+    location: [*]u8,
```

Comment:
Why the extra pointer indirection. I think we should just expose the zig sha256 directly. Cast it to an array probably so it can potentially be inline instead of requiring an allocation and extra indirection.

Reply:
On the roc side can just make it a tuple of integers that end up being sized the same as the zig type.

Just add a comptime assert on the size here. That way on zig update we can correct. I think it is currently 112 bytes before padding. So a tuple of 7 u128s should work to represent it on the roc size. On this zig side, it should be a `[7]u128 align(16)`